findPath/findPath.py

```diff
@@ -60,27 +43,27 @@ def shortestDist(start_coord: List, end_coord: List, grid: List[List]) -> int:
 # Returns:                                          #
 # Order of points which results in shortest path    #
 #####################################################
-def findPath(points: List[List], grid: List[List]) -> List[List]:
+def findPath(points: List[tuple], grid: List[List]) -> List[List]:
```

Comment:
Did you change it to tuple so you don't need to deep copy the list?

Reply:
Nope, I just changed it to match the data type given to me at Parse!

Reply:
Got it!